editor/modes/text-editor/style.scss

```diff
@@ -1,12 +1,98 @@
+.editor-text-editor__formatting {
+	background: $white;
+	border-bottom: 1px solid $light-gray-500;
+	min-height: 36px;
+
+	button {
+		height: 30px;
+		border: none;
+		background: none;
+		padding: 0 8px;
+		margin: 3px 4px;
+		text-align: center;
+		cursor: pointer;
+		font-family: $editor-html-font;
+		color: $dark-gray-500;
+		border: 1px solid transparent;
+	}
+
+	button:first-child {
+		margin-left: 0;
+	}
+
+	button:hover {
+		border: 1px solid $dark-gray-500;
+	}
+
+	button:focus {
+		outline: none;
+	}
+
+	.editor-text-editor__bold {
```

Comment:
Do these selectors need to be nested? One of the advantages of BEM is to not need excessive nesting since the names are self-describing and unique. Usually becomes an issue only when fighting for specificity.